sos/report/plugins/networking.py

```diff
@@ -183,14 +183,23 @@ def setup(self):
                 "ethtool -a " + eth,
                 "ethtool -c " + eth,
                 "ethtool -g " + eth,
-                "ethtool -e " + eth,
                 "ethtool -P " + eth,
                 "ethtool -l " + eth,
                 "ethtool --phy-statistics " + eth,
                 "ethtool --show-priv-flags " + eth,
                 "ethtool --show-eee " + eth
             ])
 
+            # skip EEPROM collection for 'bnx2x' driver NICs as this command 
+            # can pause the NIC and is not production safe.
+            cmd="ethtool -i {}".format(eth)
+            driver = self.get_command_output(cmd)
+            if not "bnx2x" in driver['output']:
+                self.add_cmd_output(["ethtool -e" + eth])
```

Comment:
`add_cmd_output()` takes either a string or list, so since this is a single command being run there's no need for it to be in a list.